packages/xo-web/src/common/xo/index.js

```diff
@@ -1918,9 +1903,18 @@ export const subscribeBackupNgJobs = createSubscription(() =>
   _call('backupNg.getAllJobs')
 )
 
-export const subscribeBackupNgLogs = createSubscription(() =>
-  _call('backupNg.getAllLogs')
-)
+export const subscribeBackupNgLogs = createSubscription(async () => {
+  const logs = { __proto__: null }
+  parseNdJson(
+    await (await fetch(
```

Comment:
Why not use `.then`?

Reply:
Because it will not make things clearer.

I've added variables maybe it will help.